crates/oq3_source_file/src/api.rs

```diff
@@ -76,9 +76,8 @@ pub fn report_error(message: &str, span: &Range<usize>, file_path: &str, source:
     let a = colors.next();
     // `offset` is a zero-indexed character offset from beginning of file.
     // Here `offset` is character index of start of error.
-    let offset = span.start;
-    Report::build(ReportKind::Error, file_path, offset)
-        //        .with_code(3)
+    Report::build(ReportKind::Error, (file_path, span.clone()))
+        .with_code(3)
```

Comment:
This seems like a behavioural change?

Reply:
I wasn't sure, I looked through the ariadne docs at the time I wrote this and I thought that it should work the same, but I might have screwed it up. It's been a busy few weeks so I don't remember the details

Reply:
Oh, I realise there's two changes highlighted.  I just meant the commented-out `.with_code` got added back.  I was assuming the other change was part of the upgrade.